.vscode/settings.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file should be added?! If not, add .vscode to gitignore

Reply:
Also part of our template